### DISCUSSION TITLE
Wrapping q-table body in q-scroll-area

### DISCUSSION THREAD

**Discussion by vindiagram** [quasarframework/quasar#14702] [General - Components / Directives / etc]
I want to have the style of the q-scroll-area for the scroll bar within the body of my q-table (Ie not wrapping top and bottom slots).
Is this possible to do?
I have attempted using the **body** slot to apply the **q-scroll-area** but it is being added to each row instead of encapsulating the entire table body.

If it is not possible to combine the **q-scroll-area** with the **q-table**, what can I do to style the **scrollbar** so it is displayed and acts the same as a **q-scroll-area**?

**Comment by MilosPaunovic** [comment]
https://codepen.io/milospaunovic/pen/ExLJqqb
Adjust CSS to your liking.

**Comment by 549393092** [comment]
Maybe you can check out this thread.
https://github.com/quasarframework/quasar/discussions/10087
But I don't know how to show the scroll bar if user put the cursor on the body of the table, hope you can figure it out.
